mlrun/config.py

```diff
@@ -104,7 +104,9 @@
     # FIXME: Adding these defaults here so we won't need to patch the "installing component" (provazio-controller) to
     #  configure this values on field systems, for newer system this will be configured correctly
     "v3io_api": "http://v3io-webapi:8081",
-    "redis_url": "redis://localhost:6379",
+    "redis": {
+        "url": "redis://localhost:6379",
```

Comment:
Is it really needed? we have the test in development and in 1.1.1 the main effort is bringing up redis - and not having this configured is only a burdon on me since I have to add it always in my code and then remove..

Reply:
removed